lib/sisimai/bite/email/facebook.rb

```diff
@@ -0,0 +1,274 @@
+module Sisimai::Bite::Email
+  module Facebook
+    # Sisimai::Bite::Email::Facebook parses a bounce email which created by Facebook.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Facebook.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->